From: Christoph Goehre <chris@sigxcpu.org>
Date: Fri, 11 Mar 2011 19:16:01 +0100
Subject: thunderbird-l10n-*: disable external extension update

All language extensions are managed by apt and can't be updated outside
the package management.

Closes: #616102
---
 icedove-l10n/ar/install.rdf    | 1 +
 icedove-l10n/ast/install.rdf   | 1 +
 icedove-l10n/be/install.rdf    | 1 +
 icedove-l10n/bg/install.rdf    | 1 +
 icedove-l10n/bn-BD/install.rdf | 1 +
 icedove-l10n/br/install.rdf    | 1 +
 icedove-l10n/ca/install.rdf    | 1 +
 icedove-l10n/cs/install.rdf    | 1 +
 icedove-l10n/cy/install.rdf    | 1 +
 icedove-l10n/da/install.rdf    | 1 +
 icedove-l10n/de/install.rdf    | 1 +
 icedove-l10n/dsb/install.rdf   | 1 +
 icedove-l10n/el/install.rdf    | 1 +
 icedove-l10n/en-GB/install.rdf | 1 +
 icedove-l10n/es-AR/install.rdf | 1 +
 icedove-l10n/es-ES/install.rdf | 1 +
 icedove-l10n/et/install.rdf    | 1 +
 icedove-l10n/eu/install.rdf    | 1 +
 icedove-l10n/fi/install.rdf    | 1 +
 icedove-l10n/fr/install.rdf    | 1 +
 icedove-l10n/fy-NL/install.rdf | 1 +
 icedove-l10n/ga-IE/install.rdf | 1 +
 icedove-l10n/gd/install.rdf    | 1 +
 icedove-l10n/gl/install.rdf    | 1 +
 icedove-l10n/he/install.rdf    | 1 +
 icedove-l10n/hr/install.rdf    | 1 +
 icedove-l10n/hsb/install.rdf   | 1 +
 icedove-l10n/hu/install.rdf    | 1 +
 icedove-l10n/hy-AM/install.rdf | 1 +
 icedove-l10n/id/install.rdf    | 1 +
 icedove-l10n/is/install.rdf    | 1 +
 icedove-l10n/it/install.rdf    | 1 +
 icedove-l10n/ja/install.rdf    | 1 +
 icedove-l10n/kab/install.rdf   | 1 +
 icedove-l10n/ko/install.rdf    | 1 +
 icedove-l10n/lt/install.rdf    | 1 +
 icedove-l10n/nb-NO/install.rdf | 1 +
 icedove-l10n/nl/install.rdf    | 1 +
 icedove-l10n/nn-NO/install.rdf | 1 +
 icedove-l10n/pa-IN/install.rdf | 1 +
 icedove-l10n/pl/install.rdf    | 1 +
 icedove-l10n/pt-BR/install.rdf | 1 +
 icedove-l10n/rm/install.rdf    | 1 +
 icedove-l10n/ro/install.rdf    | 1 +
 icedove-l10n/ru/install.rdf    | 1 +
 icedove-l10n/si/install.rdf    | 1 +
 icedove-l10n/sk/install.rdf    | 1 +
 icedove-l10n/sl/install.rdf    | 1 +
 icedove-l10n/sq/install.rdf    | 1 +
 icedove-l10n/sr/install.rdf    | 1 +
 icedove-l10n/sv-SE/install.rdf | 1 +
 icedove-l10n/ta-LK/install.rdf | 1 +
 icedove-l10n/tr/install.rdf    | 1 +
 icedove-l10n/uk/install.rdf    | 1 +
 icedove-l10n/vi/install.rdf    | 1 +
 icedove-l10n/zh-CN/install.rdf | 1 +
 icedove-l10n/zh-TW/install.rdf | 1 +
 57 files changed, 57 insertions(+)

diff --git a/icedove-l10n/ar/install.rdf b/icedove-l10n/ar/install.rdf
index 57971d3..caf461a 100644
--- a/icedove-l10n/ar/install.rdf
+++ b/icedove-l10n/ar/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/ast/install.rdf b/icedove-l10n/ast/install.rdf
index 8dd5a95..79bd47a 100644
--- a/icedove-l10n/ast/install.rdf
+++ b/icedove-l10n/ast/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/be/install.rdf b/icedove-l10n/be/install.rdf
index 77efe42..6af9124 100644
--- a/icedove-l10n/be/install.rdf
+++ b/icedove-l10n/be/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/bg/install.rdf b/icedove-l10n/bg/install.rdf
index 2a10056..eee867f 100644
--- a/icedove-l10n/bg/install.rdf
+++ b/icedove-l10n/bg/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/bn-BD/install.rdf b/icedove-l10n/bn-BD/install.rdf
index de4909e..143b1a1 100644
--- a/icedove-l10n/bn-BD/install.rdf
+++ b/icedove-l10n/bn-BD/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/br/install.rdf b/icedove-l10n/br/install.rdf
index 1636974..4dfd912 100644
--- a/icedove-l10n/br/install.rdf
+++ b/icedove-l10n/br/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/ca/install.rdf b/icedove-l10n/ca/install.rdf
index 583468e..0af77de 100644
--- a/icedove-l10n/ca/install.rdf
+++ b/icedove-l10n/ca/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/cs/install.rdf b/icedove-l10n/cs/install.rdf
index ef65a02..32d1cfd 100644
--- a/icedove-l10n/cs/install.rdf
+++ b/icedove-l10n/cs/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/cy/install.rdf b/icedove-l10n/cy/install.rdf
index e33ace1..6542a0b 100644
--- a/icedove-l10n/cy/install.rdf
+++ b/icedove-l10n/cy/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/da/install.rdf b/icedove-l10n/da/install.rdf
index 7fb4eed..038d1bf 100644
--- a/icedove-l10n/da/install.rdf
+++ b/icedove-l10n/da/install.rdf
@@ -19,5 +19,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/de/install.rdf b/icedove-l10n/de/install.rdf
index 67e1b91..b67925d 100644
--- a/icedove-l10n/de/install.rdf
+++ b/icedove-l10n/de/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/dsb/install.rdf b/icedove-l10n/dsb/install.rdf
index a0849fe..67abab4 100644
--- a/icedove-l10n/dsb/install.rdf
+++ b/icedove-l10n/dsb/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/el/install.rdf b/icedove-l10n/el/install.rdf
index 51a7444..976d52a 100644
--- a/icedove-l10n/el/install.rdf
+++ b/icedove-l10n/el/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/en-GB/install.rdf b/icedove-l10n/en-GB/install.rdf
index 34d37b2..6a0fb4d 100644
--- a/icedove-l10n/en-GB/install.rdf
+++ b/icedove-l10n/en-GB/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/es-AR/install.rdf b/icedove-l10n/es-AR/install.rdf
index 8d4d403..9c938ae 100644
--- a/icedove-l10n/es-AR/install.rdf
+++ b/icedove-l10n/es-AR/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/es-ES/install.rdf b/icedove-l10n/es-ES/install.rdf
index 1d8238c..9defd11 100644
--- a/icedove-l10n/es-ES/install.rdf
+++ b/icedove-l10n/es-ES/install.rdf
@@ -19,5 +19,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/et/install.rdf b/icedove-l10n/et/install.rdf
index eb8e83e..df410f9 100644
--- a/icedove-l10n/et/install.rdf
+++ b/icedove-l10n/et/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/eu/install.rdf b/icedove-l10n/eu/install.rdf
index 6599a73..80a1b93 100644
--- a/icedove-l10n/eu/install.rdf
+++ b/icedove-l10n/eu/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/fi/install.rdf b/icedove-l10n/fi/install.rdf
index cb7a425..a46fdff 100644
--- a/icedove-l10n/fi/install.rdf
+++ b/icedove-l10n/fi/install.rdf
@@ -19,5 +19,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/fr/install.rdf b/icedove-l10n/fr/install.rdf
index 217e725..79b649a 100644
--- a/icedove-l10n/fr/install.rdf
+++ b/icedove-l10n/fr/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/fy-NL/install.rdf b/icedove-l10n/fy-NL/install.rdf
index b4b9b60..2642d0f 100644
--- a/icedove-l10n/fy-NL/install.rdf
+++ b/icedove-l10n/fy-NL/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/ga-IE/install.rdf b/icedove-l10n/ga-IE/install.rdf
index 04b6199..4e8da93 100644
--- a/icedove-l10n/ga-IE/install.rdf
+++ b/icedove-l10n/ga-IE/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/gd/install.rdf b/icedove-l10n/gd/install.rdf
index 10a597f..e86f606 100644
--- a/icedove-l10n/gd/install.rdf
+++ b/icedove-l10n/gd/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/gl/install.rdf b/icedove-l10n/gl/install.rdf
index 9534fa2..4203757 100644
--- a/icedove-l10n/gl/install.rdf
+++ b/icedove-l10n/gl/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/he/install.rdf b/icedove-l10n/he/install.rdf
index 7256f5e..2e105ea 100644
--- a/icedove-l10n/he/install.rdf
+++ b/icedove-l10n/he/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/hr/install.rdf b/icedove-l10n/hr/install.rdf
index 93bd98d..b3fc34c 100644
--- a/icedove-l10n/hr/install.rdf
+++ b/icedove-l10n/hr/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/hsb/install.rdf b/icedove-l10n/hsb/install.rdf
index 16a131c..de6f973 100644
--- a/icedove-l10n/hsb/install.rdf
+++ b/icedove-l10n/hsb/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/hu/install.rdf b/icedove-l10n/hu/install.rdf
index da6ce12..1bb88e6 100644
--- a/icedove-l10n/hu/install.rdf
+++ b/icedove-l10n/hu/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/hy-AM/install.rdf b/icedove-l10n/hy-AM/install.rdf
index 4d98718..2cb00bd 100644
--- a/icedove-l10n/hy-AM/install.rdf
+++ b/icedove-l10n/hy-AM/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/id/install.rdf b/icedove-l10n/id/install.rdf
index e8d37c5..df4bbea 100644
--- a/icedove-l10n/id/install.rdf
+++ b/icedove-l10n/id/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/is/install.rdf b/icedove-l10n/is/install.rdf
index d94666a..5624457 100644
--- a/icedove-l10n/is/install.rdf
+++ b/icedove-l10n/is/install.rdf
@@ -19,5 +19,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/it/install.rdf b/icedove-l10n/it/install.rdf
index f76b919..e5d2b3c 100644
--- a/icedove-l10n/it/install.rdf
+++ b/icedove-l10n/it/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/ja/install.rdf b/icedove-l10n/ja/install.rdf
index 078dd43..c381fd2 100644
--- a/icedove-l10n/ja/install.rdf
+++ b/icedove-l10n/ja/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/kab/install.rdf b/icedove-l10n/kab/install.rdf
index 6c575b8..d9a5197 100644
--- a/icedove-l10n/kab/install.rdf
+++ b/icedove-l10n/kab/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/ko/install.rdf b/icedove-l10n/ko/install.rdf
index 2910e08..9399a5c 100644
--- a/icedove-l10n/ko/install.rdf
+++ b/icedove-l10n/ko/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/lt/install.rdf b/icedove-l10n/lt/install.rdf
index e0af7b2..1b12592 100644
--- a/icedove-l10n/lt/install.rdf
+++ b/icedove-l10n/lt/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/nb-NO/install.rdf b/icedove-l10n/nb-NO/install.rdf
index f437bfc..df84f8b 100644
--- a/icedove-l10n/nb-NO/install.rdf
+++ b/icedove-l10n/nb-NO/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/nl/install.rdf b/icedove-l10n/nl/install.rdf
index deb2865..f9e3c3e 100644
--- a/icedove-l10n/nl/install.rdf
+++ b/icedove-l10n/nl/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/nn-NO/install.rdf b/icedove-l10n/nn-NO/install.rdf
index da2decf..830738c 100644
--- a/icedove-l10n/nn-NO/install.rdf
+++ b/icedove-l10n/nn-NO/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/pa-IN/install.rdf b/icedove-l10n/pa-IN/install.rdf
index 25b7c3e..0f3c8a1 100644
--- a/icedove-l10n/pa-IN/install.rdf
+++ b/icedove-l10n/pa-IN/install.rdf
@@ -19,5 +19,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/pl/install.rdf b/icedove-l10n/pl/install.rdf
index 46abffd..ed69433 100644
--- a/icedove-l10n/pl/install.rdf
+++ b/icedove-l10n/pl/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/pt-BR/install.rdf b/icedove-l10n/pt-BR/install.rdf
index 6e159b3..6a8d120 100644
--- a/icedove-l10n/pt-BR/install.rdf
+++ b/icedove-l10n/pt-BR/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/rm/install.rdf b/icedove-l10n/rm/install.rdf
index 3ce7789..294cee5 100644
--- a/icedove-l10n/rm/install.rdf
+++ b/icedove-l10n/rm/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/ro/install.rdf b/icedove-l10n/ro/install.rdf
index d9e1f71..4ef588b 100644
--- a/icedove-l10n/ro/install.rdf
+++ b/icedove-l10n/ro/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/ru/install.rdf b/icedove-l10n/ru/install.rdf
index cedfb9b..449dff2 100644
--- a/icedove-l10n/ru/install.rdf
+++ b/icedove-l10n/ru/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/si/install.rdf b/icedove-l10n/si/install.rdf
index 1dbee3b..99df876 100644
--- a/icedove-l10n/si/install.rdf
+++ b/icedove-l10n/si/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/sk/install.rdf b/icedove-l10n/sk/install.rdf
index 89b6c2b..9bad278 100644
--- a/icedove-l10n/sk/install.rdf
+++ b/icedove-l10n/sk/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/sl/install.rdf b/icedove-l10n/sl/install.rdf
index cb8d7c1..6c63607 100644
--- a/icedove-l10n/sl/install.rdf
+++ b/icedove-l10n/sl/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/sq/install.rdf b/icedove-l10n/sq/install.rdf
index a102288..0413a8c 100644
--- a/icedove-l10n/sq/install.rdf
+++ b/icedove-l10n/sq/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/sr/install.rdf b/icedove-l10n/sr/install.rdf
index 80d4e88..dae2f3d 100644
--- a/icedove-l10n/sr/install.rdf
+++ b/icedove-l10n/sr/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/sv-SE/install.rdf b/icedove-l10n/sv-SE/install.rdf
index 2f5cf2c..844a740 100644
--- a/icedove-l10n/sv-SE/install.rdf
+++ b/icedove-l10n/sv-SE/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/ta-LK/install.rdf b/icedove-l10n/ta-LK/install.rdf
index a269056..017846b 100644
--- a/icedove-l10n/ta-LK/install.rdf
+++ b/icedove-l10n/ta-LK/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/tr/install.rdf b/icedove-l10n/tr/install.rdf
index 525a40d..197ecda 100644
--- a/icedove-l10n/tr/install.rdf
+++ b/icedove-l10n/tr/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/uk/install.rdf b/icedove-l10n/uk/install.rdf
index 7b48b44..937a853 100644
--- a/icedove-l10n/uk/install.rdf
+++ b/icedove-l10n/uk/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/vi/install.rdf b/icedove-l10n/vi/install.rdf
index 11f7671..95caecc 100644
--- a/icedove-l10n/vi/install.rdf
+++ b/icedove-l10n/vi/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/zh-CN/install.rdf b/icedove-l10n/zh-CN/install.rdf
index 63a5437..e90345a 100644
--- a/icedove-l10n/zh-CN/install.rdf
+++ b/icedove-l10n/zh-CN/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
diff --git a/icedove-l10n/zh-TW/install.rdf b/icedove-l10n/zh-TW/install.rdf
index 71fd866..33cb35c 100644
--- a/icedove-l10n/zh-TW/install.rdf
+++ b/icedove-l10n/zh-TW/install.rdf
@@ -20,5 +20,6 @@
         <em:maxVersion>52.*</em:maxVersion>
       </Description>
     </em:targetApplication>
+    <em:appManaged>true</em:appManaged>
   </Description>
 </RDF>
